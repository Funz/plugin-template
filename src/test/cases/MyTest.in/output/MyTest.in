sample input file here
... $x ...
... u=@{ $x + 1 } ...
...
print('z',1.23456)
...