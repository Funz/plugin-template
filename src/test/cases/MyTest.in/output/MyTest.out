output information here
...
z= 3.000715 
...